src/hardware/developing-an-iox/protocol.md

```diff
@@ -281,6 +281,18 @@ Sent from the IOX to the GO device to inform the GO device of events or status c
 | 0-1 | 0x0004 |
 | 2 | Unused |
 
+#### Information Type 11 - IOX Third-Party Tx-Buffer State (GO9 with WIFI only)
+The IOX can choose to send a "Stop-sending" message to GO for every Tx Data(0x0B) message it receives when the IOX believes its internal buffer is about to be overrun.<br>
+When the GO receives the  "Stop-sending" message, it should pause sending messages out from the GO Tx-message queue to the IOX.<br>
+When the IOX internal buffer returns to the level that it can accept more messages, the IOX will send out a "Resume-sending" message to the GO.<br>
```

Comment:
"IOX will send out", suggest changing to "IOX should send out".

Reply:
Fixed.